### PR TITLE
[MAINT] Fix FutureWarning in data.py

### LINKED ISSUE #4: MAINT: Fix FutureWarning in data.py
Reported in Issue [#170](https://github.com/QuantEcon/lecture-python-intro/issues/170).

### PR DESCRIPTION
**Pull request by HumphreyYang**
Hi @jstac,

This PR fixes #4. Please kindly review and merge if it looks good to you.

Many thanks in advance.

### CODE REVIEW

**Comment by jstac** [comment]
Thanks @HumphreyYang .

@HumphreyYang could you please review this for me and check that it preserves the same output and removes the future warning?

**Comment by HumphreyYang** [comment]
Hi @jstac,

I did this check last night -- you can check the result [here](https://gist.github.com/HumphreyYang/43ece4c12e3ed1e85aafa210e7781ee8)

**Comment by jstac** [comment]
Thanks @HumphreyYang .  I appreciate that.

Another set of eyes passing over it is still a good idea.

File: quantecon_book_networks/data.py [hunk 1]
```diff
@@ -108,9 +108,9 @@ def introduction():
     data_file = pkg_resources.resource_stream(__name__, data_file)
     crude_oil = pd.read_csv(data_file, dtype={'product_id': str})
 
-    exporters = crude_oil.groupby(by=["location_code"]).sum().sort_values("export_value", ascending=False)[:10].index
+    exporters = crude_oil.groupby(by=["location_code"]).sum(numeric_only=True).sort_values("export_value", ascending=False)[:10].index
```
Comment (mmcky) [inline quantecon_book_networks/data.py]: @HumphreyYang from a `data person` perspective this makes me a little nervous. I would select the columns that are `value` columns via selection in the `crude_oil` data frame such as `crude_oil['export_value'].groupby(by ....`

Comment (HumphreyYang) [reply quantecon_book_networks/data.py]: Hi @mmcky, 

I have committed to the requested change. Please kindly merge if it looks good to you. I will double-check if the future warning is resolved in the intro lecture once this is merged. 

Many thanks in advance.

**Comment by HengchengZhang** [comment]
Hi @mmcky, @HumphreyYang told me about this error exists in the latest `v0.5` version.

I've tested the changes Humphrey made here in the latest code and they work well.

The thing is that I can see there's conflicts between this change and `main` branch.

My suggestion is that shall we merge this one into `fix_name` branch and I can resolve the conflicts there and then we merge these changes together.

**Comment by HumphreyYang** [comment]
Hi @HengchengZhang,

Could you resolve this conflict here? We are building previews now, so please kindly resolve it now if possible.

Many thanks!

**Comment by HengchengZhang** [comment]
Sure thing

**Comment by HengchengZhang** [comment]
Hi @HumphreyYang, since you are merging from your own fork I can't make changes in your own branch.

I've resolved the conflicts in a new branch and I hope it works.

**Comment by mmcky** [comment]
> Hi @HumphreyYang, since you are merging from your own fork I can't make changes in your own branch.
> 
> I've resolved the conflicts in a new branch and I hope it works.

@HengchengZhang if you install the `GitHub` cli https://github.com/cli/cli then you can

```
gh pr checkout <number>
<make changes>
git push
```

This will push to the fork and setup all the origins for you.

**Pull request merged**